sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rule/BeamAggregationRule.java

```diff
@@ -49,9 +49,7 @@ public void onMatch(RelOptRuleCall call) {
     final Aggregate aggregate = call.rel(0);
     final Project project = call.rel(1);
     RelNode x = updateWindow(call, aggregate, project);
-    if (x != null) {
-      call.transformTo(x);
-    }
+    call.transformTo(x);
```

Comment:
How do we know this x is not null here?

Reply:
As "updateWindow" returns "return new BeamAggregationRel(..."?